app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Here are the details for given user</h1>
\ No newline at end of file
+<div>
+  <%= render partial: 'users/user_info', locals: { user: @user } %>
+</div>
+
+<div class="profile-bio-div">
+  <p class="profile-bio-tag">Bio</p>
+  <p class="profile-bio"><%= @user.bio %></p>
+</div>
+
+<div>
+  <% @user.posts.each do |post| %>
+    <div>
+      <!-- Render the post_info partial -->
+      <%= render partial: 'posts/post_info',
+      locals: {post: post, user: @user, comments: post.comments} %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] @mershark Great work displaying posts in the user show page, however, you are supposed to use the `most_recent_posts`  method to display the three latest posts only in this page.